<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(43, 51, 62)</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
            border-image:url(:/buttonbackground.png);
            color:white;
        }
        QPushButton:hover{
            border-image:url(:/buttonbackground.png);
        color:white;
        }
        QPushButton:pressed{
            border-image:url(:/button.png);
        color:white;
        };
</string>
    </property>
    <property name="text">
     <string>资讯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
            border-image:url(:/buttonbackground.png);
            color:white;
        }
        QPushButton:hover{
            border-image:url(:/buttonbackground.png);
        color:white;
        }
        QPushButton:pressed{
            border-image:url(:/button.png);
        color:white;
        };
</string>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
            border-image:url(:/buttonbackground.png);
            color:white;
        }
        QPushButton:hover{
            border-image:url(:/buttonbackground.png);
        color:white;
        }
        QPushButton:pressed{
            border-image:url(:/button.png);
        color:white;
        };
</string>
    </property>
    <property name="text">
     <string>单词本</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>45</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Verdana&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(51, 61, 73)</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255);
border:none;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>45</width>
       <height>10</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>UserName</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(51, 61, 73)</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 75 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>悟词典</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(234, 239, 243)</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>286</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:rgb(51, 61, 73);
color:white;</string>
     </property>
     <property name="text">
      <string>六级单词一键导入！</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>精彩阅读</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>12</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border:none;</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <widget class="QGraphicsView" name="graphicsView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>164</x>
        <y>12</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border:none;</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>致胜单词</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;
color:black;</string>
     </property>
     <property name="text">
      <string>查找单词</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:white;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>381</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:white;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>单词本</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>411</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;
border:none;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>401</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 30pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Word</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>Answer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>289</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(43, 51, 62);
border:none;
color:white;
font: 75 11pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>加入我的单词本</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255)</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 239, 243);
border:none;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
            background:rgb(43, 51, 62);
			border:none;
            color:white;
        }
        QPushButton:hover{
            background:rgb(43, 51, 62);
        color:white;
        }
        QPushButton:pressed{
            background:rgb(34, 40, 52);
        color:white;
        };</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-image: url(:/close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
            
	       border-image: url(:/info.png);
        }
        QPushButton:hover{
            
	         border-image: url(:/info.png);
        }
        QPushButton:pressed{
            
	border-image: url(:/info.png);

        }
        QPushButton::menu-indicator{
            image:none;
        };
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
